stackslib/src/chainstate/nakamoto/test_signers.rs

```diff
@@ -0,0 +1,195 @@
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
```

Comment:
If this is all test-only code, then please put it into `stackslib/src/chainstate/nakamoto/tests/test_signers.rs`.

Reply:
It isn't -- it's used by mockamoto and the self signing behavior of the stacks-node.